src/components/Table/nestedCards.test.tsx

```diff
@@ -0,0 +1,193 @@
+import React from "react";
+import { NestedCardStyle } from "src/components/Table/GridTable";
+import { makeOpenOrCloseCard, maybeAddCardPadding } from "src/components/Table/nestedCards";
+import { Palette } from "src/Css";
+import { render } from "src/utils/rtl";
+
+const parentCardStyle: NestedCardStyle = {
+  bgColor: Palette.Gray100,
+  brPx: 8,
+  pxPx: 8,
+  spacerPx: 8,
+};
+
+const childCardStyle: NestedCardStyle = {
+  bgColor: Palette.Gray300,
+  bColor: Palette.Gray200,
+  brPx: 6,
+  pxPx: 6,
+  spacerPx: 6,
+};
+
+describe("nestedCards", () => {
+  it("can make open card w/one level", async () => {
+    const r = await render(makeOpenOrCloseCard([parentCardStyle], "open"));
+    expect(r.firstElement).toMatchInlineSnapshot(`
```

Comment:
**thought:** I would push to have these "tests" be storybook snapshots instead of jest. Just seeing the styles and minimal html output isn't immediately clear on what this is accomplishing/validating. Without seeing the changes visually it is kind of a difficult test to glance over and understand.